copulas/multivariate/tree.py

```diff
@@ -495,14 +495,14 @@ def _identify_eds_ing(first, second):
                 The first two values represent left and right node
                 indicies of the new edge. The third value is the new dependence set.
         """
-        A = set([first.L, first.R])
+        A = {first.L, first.R}
         A.update(first.D)
 
-        B = set([second.L, second.R])
+        B = {second.L, second.R}
         B.update(second.D)
 
         depend_set = A & B
-        left, right = sorted(list(A ^ B))
+        left, right = sorted(list(A ^ B))  # noqa: C414
```

Comment:
Is the `noqa` really needed? In other words, could this just be `sorted(A ^ B)` without the `list`?

Reply:
You are correct 👍